### PR TITLE
Sprint3 vgnwb 104

### PR DESCRIPTION
**Pull request by kardanitin**
Added changes to make fullName, Affiliation, Location mandatory fields.

### CODE REVIEW

File: annotations/forms.py [hunk 1]
```diff
@@ -27,10 +27,14 @@ class RegistrationForm(forms.Form):
 	"""
 	Gives user form of signup and validates it.
 	"""
+	full_name = forms.CharField(widget=forms.TextInput(attrs=dict(required=True, max_length=30, render_value=False)), label=_("Full name"))
```
Comment (erickpeirson) [inline annotations/forms.py]: Yikes, this adds a whole lot of code! Do you really need to explicitly set the widgets on all of these fields? The defaults are usually enough. If you're worried about labels, there are other (easier) places to change those.

Also, don't let your lines exceed 79 characters. ;-)

File: annotations/models.py [hunk 2]
```diff
@@ -11,13 +11,16 @@
 
 
 class VogonUserManager(BaseUserManager):
-    def create_user(self, username, email, password=None):
+    def create_user(self, username, email, password=None,full_name=None,affiliation=None,location=None):
```
Comment (erickpeirson) [inline annotations/models.py]: Leave a space after every comma

**Comment by erickpeirson** [comment]
Great start! See my comments, above. In general, try to write as little code as possible: every bit of code that you write has to be maintained.

**Comment by kardanitin** [comment]
Incorporated Suggestions provided

File: annotations/forms.py [hunk 3]
```diff
@@ -27,10 +27,19 @@ class RegistrationForm(forms.Form):
...
+                                                              "numbers and underscores.") })
 	email = forms.EmailField(widget=forms.TextInput(attrs=dict(required=True, max_length=30)), label=_("Email address"))
-	password1 = forms.CharField(widget=forms.PasswordInput(attrs=dict(required=True, max_length=30, render_value=False)), label=_("Password"))
-	password2 = forms.CharField(widget=forms.PasswordInput(attrs=dict(required=True, max_length=30, render_value=False)), label=_("Password (again)"))
+	password1 = forms.CharField(widget=forms.PasswordInput(attrs=dict(required=True, max_length=30,
+                                                                      render_value=False)), label=_("Password"))
+	password2 = forms.CharField(widget=forms.PasswordInput(attrs=dict(required=True, max_length=30,
+                                                                      render_value=False)), label=_("Password (again)"))
+	affiliation = forms.CharField(required=True, max_length=30, label=_("Location"))
+	location = forms.CharField(required=False, max_length=30, label=_("Affliation"))
```
Comment (erickpeirson) [inline annotations/forms.py]: Shouldn't location be required?

Comment (kardanitin) [inline annotations/forms.py]: We can make it mandatory if required

Comment (erickpeirson) [inline annotations/forms.py]: Yes please :-) 

"When a user registers, full name, affiliation, and **location** should be mandatory fields"

Comment (kardanitin) [inline annotations/forms.py]: Done

**Comment by erickpeirson** [comment]
Word. Thanks!

**Pull request merged**